Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'http://rubygems.org'
 
-ruby '1.9.3', engine: 'jruby', engine_version: '1.7.4'
+ruby '1.9.3'  # , engine: 'jruby', engine_version: '1.7.4'
```

Comment:
This is not correct. It only coincidentally works on MRI if you're not using the PDF conversions.